src/content/getusermedia/gum/js/main.js

```diff
@@ -8,23 +8,22 @@
 
 'use strict';
 
-// variables in global scope so available to console
-var video = document.querySelector('video');
-var constraints = {
+// Put variables in global scope to make them available to the browser console.
+var video = document.getElementById('gum-local');
```

Comment:
Same here: I prefer `querySelector()` in these examples, with the element name for clarity (though I think csslint may complain, because of the performance hit, though that's not relevant here).